app/models/user.rb

```diff
@@ -42,7 +42,7 @@ class User < ApplicationRecord
                    length: { minimum: 2, maximum: 255 } # TODO: amir - Change into full_name or seperate first and last name.
 
   validates :email,
-            format: /\A[\w\-.]+@[\w\-.]+\.[a-z]+\z/i,
+            format: /\A[\w\-.+]+@[a-z\d-]+(\.[a-z\d-]+)*\.[a-z]+\z/i,
```

Comment:
Just adding the "+" in the first list seems to work:
\A[\w\-.+]+@[\w\-.]+\.[a-z]+\z